opentelemetry-api/setup.py

```diff
@@ -45,7 +45,7 @@
     ],
     extras_require={},
     license="Apache-2.0",
-    packages=setuptools.find_namespace_packages(include=["opentelemetry.*"]),
+    packages=setuptools.find_namespace_packages(include=["opentelemetry", "opentelemetry.*"]),
```

Comment:
This should have been solved in #37, please rebase the latest master branch.

Reply:
Done.